contracts/axelar-interchain-token-service/src/test.rs

```diff
@@ -108,101 +100,44 @@ fn set_trusted_address_fails_if_not_owner() {
 
 #[test]
 fn set_trusted_address_fails_if_already_set() {
-    let (env, contract_id, client) = setup_env();
+    let (env, _contract_id, client) = setup_env();
```

Comment:
```suggestion
    let (env, _, client) = setup_env();
```
perhaps

Reply:
Do you not find the `_ignore` pattern potentially useful, say for example if later on the test changes and we need to use the `contract_id`?